templates/backup-pruner.sh

```diff
@@ -0,0 +1,14 @@
+#jinja2: trim_blocks:False
+#!/bin/bash
+# This script runs all of the backup pruners in parallel, returning the output
+# of only the failing pruners.
+# Check {{ JOB_LOG }} for a summary of the most recent pruner run.
+set -e
+
+jobs='{% for item in TARSNAPPER_JOBS %}
+{{ item.name }}{% endfor %}'
+
+# Buffer all of the pruner output; if the pruner fails, forward along the
+# output to include in a cron email.
+wrapper='output=$(/usr/local/sbin/tarsnap-{}.sh expire 2>&1) || echo "$output"'
+echo "$jobs" | tail -n +2 | parallel --no-notice -j0 --joblog {{ JOB_LOG }} $wrapper
```

Comment:
Parallel does write out a nice little log containing the exit code and total running time of each of the subprocesses, but I found the tarsnapper output is usually pretty good at saying exactly what went wrong, so I wanted to make sure the cron emails we receive contain the output for any failing tarsnapper runs so that we'll know immediately how serious the problem is.

To accomplish this I just wrapped the task in a form like this:
```
output=$(task_to_execute 2>&1) || echo "$output"
```
This just buffers the output into `output` until the task finishes executing. Then, only if the return code is non-zero, it'll forward along the output to Parallel.
Parallel doesn't touch the output, it just buffers the output and forwards it along when the task completes as well, which cron will then receive and trigger the email.

So, in a situation where none of the pruners fail, all of the output will be discarded and no email will be triggered.

Little wrappers like `cronic` provide this sort of behavior, but `cronic` in particular also treats any output to `stderr` as if it's worthy of triggering an email. Tarsnapper always writes some output to `stderr`, and we only care about the exit code here, so I did this instead.

Let me know if you have any suggestions to improve the readability here. I agree that it's not very easy to understand, although I hoped my comment would help.